products.js

```diff
@@ -18,16 +18,16 @@ const {dom, props, out, rule, ruleset, score, type} = require('fathom-web');
 const {Annealer} = require('fathom-web/optimizers');
 const {staticDom} = require('fathom-web/utils');
 const tuningRoutines = {
-                        'title' : {'routine': tunedTitleFnodes, 'coeffs': []},
-                        'price' : {'routine': tunedPriceFnodes, 'coeffs':   [ 1.6, 1, 0.7, 0.7, 1.9000000000000001, 0.39999999999999997, 1, 1.3, 0.7, 1.3, 1.3, 1.3, 0.09999999999999998, 1, 0.39999999999999997, 0.39999999999999997, 0.7, 1.9000000000000001, 7.3 ]},
-                        'image' : {'routine': tunedImageFnodes, 'coeffs': [1.9, 3.0, 4.2, 0.5, 0.1, 0.8, 1.3, 0.7, 0.4, 1, 0.1, 0.1, 1.3]}
+                        // 'title' : {'routine': tunedTitleFnodes, 'coeffs': []},
```

Comment:
Do you mean to comment these in and out on every commit? There's a lot of thrash. I suspect you are not reading your diffs before committing. :-o ;-)